src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 300;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.search-form__label {
+  display: flex;
+  /* justify-content: center;
+  align-items: center; */
+  position: relative;
+}
+
+.search-form__input {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+  padding-left: 62px;
+  width: 100%;
+  height: 66px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: inherit;
+  align-items: center;
+}
+
+.search-form--input-small {
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  padding-left: 33px;
+  width: 100%;
+  height: 38px;
+}
+
+.search-form__input:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
```

Comment:
color `#e1e7ed` is duplicated, so please move all duplicated styles to the variable